src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -16,13 +17,12 @@ import scala.concurrent.duration._
 object SubmissionSupervisor {
   sealed trait SubmissionSupervisorMessage
 
-  case class SubmissionStarted(workspaceName: WorkspaceName, submissionId: String, credential: Credential)
+  case class SubmissionStarted(workspaceName: WorkspaceName, submissionId: UUID, credential: Credential)
 
   def props(executionServiceDAO: ExecutionServiceDAO,
             datasource: SlickDataSource,
-            workflowPollInterval: Duration = 1 minutes,
-            submissionPollInterval: Duration = 30 minutes): Props = {
-    Props(new SubmissionSupervisor(executionServiceDAO, datasource, workflowPollInterval, submissionPollInterval))
+            submissionPollInterval: FiniteDuration = 1 minutes): Props = {
```

Comment:
is it worth pulling the duration out into config, so we can tune it per-environment? (brainstorming here, not suggesting)